cuckoo/core/guest.py

```diff
@@ -460,9 +463,15 @@ def start_analysis(self, options, monitor):
 
         # If the target is a file, upload it to the guest.
         if options["category"] == "file" or options["category"] == "archive":
+
+            if self.platform in ("linux", "darwin"):
```

Comment:
I find `if self.platform == "linux" or self.platform == "darwin":` to be slightly neater in terms of code syntax.